osgi/jta/pom.xml

```diff
@@ -75,6 +75,12 @@
 		   <artifactId>jboss-transaction-spi</artifactId>
 		   <version>${version.org.jboss.jboss-transaction-spi}</version>
 		   <scope>provided</scope>
+       <exclusions>
+           <exclusion>
+               <groupId>org.jboss.spec.javax.transaction</groupId>
+               <artifactId>jboss-transaction-api_1.1_spec</artifactId>
+           </exclusion>
+         </exclusions>
```

Comment:
Either that or there is a bug in the jboss-transaction-spi where it is somehow pulling in JTA 1.1 and JTA 1.2